src/main/java/com/scalar/cassy/placer/Placer.java

```diff
@@ -38,13 +38,14 @@ public void place(RestoreConfig config) {
   private void place(List<Path> files, Path fromDir, Path toDir, int dirToFileDistance) {
```

Comment:
By https://github.com/scalar-labs/cassy/pull/45/files#r404051722 ,
`files` are now full path. (previously it is a relative path)